sql/common/src/main/scala/org/apache/spark/sql/sedona_sql/expressions/st_functions.scala

```diff
@@ -275,4 +275,14 @@ object st_functions extends DataFrameAPI {
 
   def ST_ZMin(geometry: Column): Column = wrapExpression[ST_ZMin](geometry)
   def ST_ZMin(geometry: String): Column = wrapExpression[ST_ZMin](geometry)
+
+  def ST_GeometricMedian(geometry: Column): Column = wrapExpression[ST_GeometricMedian](geometry, 1e-6, 1000, false)
```

Comment:
Please also add DataFrame Style API to Sedona Python. See examples: https://github.com/apache/sedona/blob/master/python/sedona/sql/st_functions.py#L882

You don't need to perform tests in Sedona Python if you don't want.